src/components/controls/time/Time.js

```diff
@@ -25,7 +25,7 @@ var Time = React.createClass({
      * @return {string}         Timestamp in the format of HH:MM:SS
      */
     formatTime(seconds) {
-        var date = new Date(null);
+        var date = new Date(Date.UTC(2000,1,1,0,0,0,0);
```

Comment:
I think it was some temporary problem, the issue was gone when I tried again after lunch.
I corrected the typo and changed the date to 1970, however, the day must be 1, not 0